pkg/apis/pravega/v1beta1/pravega.go

```diff
@@ -235,6 +315,32 @@ type PravegaSpec struct {
 	// InfluxDB Secret specifies the secret name containing credentials and mount path volume
 	// that has to be configured in controller and segmentstore pods
 	InfluxDBSecret *InfluxDBSecret `json:"influxDBSecret,omitempty"`
+
+	// Probes specifies the timeout values for the Readiness and Liveness Probes
+	// for the controller pods.
+	ControllerProbes *Probes `json:"controllerProbes,omitempty"`
+
+	// Probes specifies the timeout values for the Readiness and Liveness Probes
```

Comment:
Same as above :
`SegmentStore Probes specifies the values for configurable fields of Readiness and Liveness Probes for the segmentstore pods`